### ISSUE TITLE
PIP resize button location glitch

### ISSUE THREAD

**Issue by peteruithoven** [elementary/gala#395]
The PIP resize button seems to move itself after the first mouse down. The draggable area however stays in the same area.
![peek 2018-10-27 03-11](https://user-images.githubusercontent.com/523210/47598000-1080d300-d996-11e8-8195-9ddf68a57fde.gif)

**Comment by JosephMcc** [comment]
Yeah, I remember seeing this during the beta. Was going to try and fix it but ran out of time. I do believe it only happens under hidpi.
